iac/acm-multienv-cicd-anthos-autopilot/base/config.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
```

Comment:
I propose to 'move' these files instead of 'copying' them. Since this base configuration is a kustomize component, it can be easily added to a skaffold configuration. As discussed I have prepared a skaffold profile ('development') which is compatible with a vanilla k8s cluster for the next PR that will include this. 

Source to include these manifests specifically can be found [here](https://github.com/aablsk/bank-of-anthos/blob/94777354f75cc90c7e19da45f0e640d1b40b3bac/skaffold.yaml#L14-L24).